static/app/views/dashboardsV2/controls.tsx

```diff
@@ -139,6 +139,16 @@ class Controls extends React.Component<Props> {
             }}
           />
         </DashboardSelect>
+        <Feature features={['organizations:dashboards-manage']}>
+          <Button
+            data-test-id="dashboard-manage"
+            to={{
+              pathname: `/organizations/${organization.slug}/dashboards/manage/`,
```

Comment:
Looks like we are not retaining global selection when switching between dashboards- so I'll just leave it as is for now for consistency. But if we are coming from outside the scope of Dashboards (like Performance -> Dashboards), I have it persist, but that could feel like an inconsistent experience. I'm going to think about this.